ChunkCodecCore/src/public.jl

```diff
@@ -0,0 +1,26 @@
+@static if VERSION ≥ v"1.11.0-DEV.469"
+    macro public(s::Symbol)
+        return esc(Expr(:public, s))
+    end
+    macro public(e::Expr)
+        return esc(Expr(:public, e.args...))
+    end
+else
+    macro public(e) end
```

Comment:
```suggestion
    macro var"public"(s::Symbol)
        return esc(Expr(:public, s))
    end
    macro var"public"(e::Expr)
        return esc(Expr(:public, e.args...))
    end
else
    macro var"public"(e) end
```
Public is a keyword so it is best to protect it with `var`. (the same way you would have to write an `@export` macro).